src/cloud/project/project-patch.md

```diff
@@ -64,6 +64,12 @@ To apply and test a custom patch:
 
 1. Copy the patch file to the `/m2-hotfixes` directory.
 
+1. Add, commit, and push your code changes:
+
+   ```bash
+   git add -A && git commit -m "Apply <patch filename> patch" && git push origin
```

Comment:
Sometimes this works, but sometimes it gives errors to string the commands together. And you have a var in the message. Probably best to keep that generic. "Apply patch"